src/main/asciidoc/_chapters/developer.adoc

```diff
@@ -1783,6 +1783,25 @@ We use Git for source code management and latest development happens on `master`
 branches for past major/minor/maintenance releases and important features and bug fixes are often
  back-ported to them.
 
+=== Policy for Fix Version in JIRA
+
+Currently branch-1 points to branch-1.5 as the upcoming HBase 1.x release. The policy for Fix
```

Comment:
Remove the first sentence. Otherwise it needs to be changed with upcoming releases.